article/app/controllers/LiveBlogController.scala

```diff
@@ -1,6 +1,7 @@
 package controllers
 
 import com.gu.contentapi.client.model.v1.{Blocks, ItemResponse, Content => ApiContent}
+import com.gu.contentapi.client.utils.format.NewsPillar
```

Comment:
We may need to import SportPillar too?

I wonder if there are some unused imports being added further down in this file, which might be worth checking (this is according to intellij though, so could be wrong!)

Reply:
+1

Reply:
SportPillar is already imported on line 5, though the imports could be on the same line.

Reply:
Oh hey Joe, sorry I missed this (and then just merged through). For my understanding (and general learning of Scala) how would I bring the two imports into one line?